### ISSUE TITLE
pretrained model

### ISSUE THREAD

**Issue by mzarvandi** [Goldesel23/Siamese-Networks-for-One-Shot-Learning#8]
Hi,
I tried to train the network with a server using two GPUs still it's so much time consuming.
Could you please share the weights and biases for pretrained network.

Thank you

**Comment by tensorfreitas** [comment]
Hi Maedeh, 

I do not have the pre-trained model weights to share, but it should take some hours to train the model.

Best Regards
Tiago Freitas

**Comment by mzarvandi** [comment]
HI Tiago,

I see. But I also configured that before 18 thousands epochs it doesn't
reach to any better accuracy so it stops training, and by the way the best
accuracy during test it reaches is 0.6125. I was wondering if you could
tell me how to reach to a better accuracy. I also tried Adam optimizer it
did not go further than about 70 percent accuracy. I will be thankful if
you  could give me a hint.

Regards,

**Comment by tensorfreitas** [comment]
Hi Maedeh,

Sorry for the late response, 

Are you training on Omniglot dataset with the exact same code/architecture?
